test/image/mocks/annotations.json

```diff
@@ -17,6 +17,7 @@
             {"text":"right bottom","showarrow":false,"xref":"paper","yref":"paper","xanchor":"right","yanchor":"bottom","x":0.5,"y":1},
             {"text":"move with page","xref":"paper","yref":"paper","x":0.75,"y":1},
             {"text":"opacity","opacity":0.5,"x":5,"y":5},
+            {"text":"not-visible", "visible": false},
```

Comment:
> do you want to also include an invisible annotation in annotations-autorange.json

done in a2dd6347f0ba9b4fd8b8741a6e61dc06642dff17

> Same for shapes.

the [`shape_below_traces`](https://github.com/plotly/plotly.js/blob/master/test/image/mocks/shapes_below_traces.json) is autoranged, so that's a :white_check_mark: already

> I guess images currently don't contribute to autorange at all - is that on purpose or should I consider it a bug?

I'd say that's a :bug: